src/tsickle.ts

```diff
@@ -1897,3 +1899,213 @@ export function getGeneratedExterns(externs: {[fileName: string]: string}): stri
   }
   return allExterns;
 }
+
+export interface TransformerHost extends es5processor.Es5ProcessorHost, AnnotatorHost {
```

Comment:
You could rename this back into `TsickleHost`..

Reply:
Done.